aclk/agent_cloud_link.c

```diff
@@ -173,20 +222,26 @@ struct aclk_query *aclk_query_find_position(time_t time_to_run)
 }
 
 // Need to have a lock before calling this
-struct aclk_query *aclk_query_find(char *topic, char *data, char *msg_id, char *query)
+struct aclk_query *aclk_query_find(char *topic, char *data, char *msg_id, char *query, ACLK_CMD cmd, struct aclk_query **last_query)
```

Comment:
Every function in this file that is not called from elsewhere should be static.

Reply:
Will add in future issue